polyfills/Number/parseInt/config.json

```diff
@@ -0,0 +1,22 @@
+{
+	"aliases": [
+		"es6"
+	],
+	"browsers": {
+		"ie": "<12",
+		"ie_mob" : "*",
+		"firefox": "<34",
```

Comment:
I mixed Chrome and firefox. Both were wrong. Fixed now. tested in browserstack to be super sure.